### PR TITLE
General UI rework

### PR DESCRIPTION
**Pull request by reesque**
- Reworked Summary screen to be side by side, left to right
- Added TTS to summary screen
- Added dummy endpoints to server, and switch to dummy endpoints in code for dev purpose
- Reworked buttons to be more modular and have hover effects

### CODE REVIEW

File: frontend/src/assets/submit-plane.png
Comment (hungnguyendinh1999) [inline frontend/src/assets/submit-plane.png:1]: I don't see any difference

Comment (reesque) [reply frontend/src/assets/submit-plane.png:1]: Due to multiple recolor

**Review by hungnguyendinh1999 — APPROVED** [review]
LGTM

**Review by ryqui — APPROVED** [review]
looks a-ok

**Pull request merged**